source/codegen/metadata/nirfmxinstr/functions.py

```diff
@@ -1500,6 +1500,8 @@
         'returns': 'int32'
     },
     'GetNIRFSASessionArray': {
+        'codegen_method': 'CustomCode',
```

Comment:
Will this go into the metadata extraction repo after these changes go in here? Has that been the process for keeping the versions of the metadata in sync? Get manual updates in in grpc-device and then in scrapigen?

Reply:
Yes, that will need to go in. I'll leave this comment open until this is up.

Reply:
Oh, you asked more questions, sorry.

> Has that been the process for keeping the versions of the metadata in sync? 

I try to do them "about the same time". When the codgen is new/complicated, the grpc-device part tends to go first (partly to vet the PR). When the change is mostly extraction, sometimes scrapigen goes first. We'll need to be more disciplined about keeping them in sync with more people working on it.

@DavidCurtiss has a script to sync them for a local dev workflow. We can consider making that a workflow on scrapigen if we can figure out how to make it work. I guess from a workflow we can clone another repo, copy over rfmx metadata, and then check for diffs. We could make that non-blocking, but we'd want to trigger on changes either direction (and we'd expect it to be broken fairly frequently; but hopefully settle to "green" pretty quickly too,.

Reply:
I do have a simple python script for copying all of the rfmx dirs (and I can share that if someone would find it useful), but mostly I just use my NR-specific .bat file, since that's all I'm normally editing:
```
@mkdir ..\grpc-device\source\codegen\metadata\nirfmxnr\ 2>NUL
cp -u out/rfmxnr/export/* ../grpc-device/source/codegen/metadata/nirfmxnr/
```
P.S. I also have .bat files for "building" (generating) and testing, so I don't have to think about the commands. More useful, though, are my slightly-more-complicated .bat files for grpc-device, which I've set up to run my ninja-based builds.